__docs__/components/gallery/tiles/breadcrumbs-tile.tsx

```diff
@@ -25,12 +25,14 @@ export default function BreadcrumbsTile(props: CommonTileProps) {
             <View style={styles.centerContent}>
                 <Breadcrumbs>
                     <BreadcrumbsItem>
-                        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid -- TODO: Address a11y error */}
-                        <Link href="">Course</Link>
+                        <Link href="#course" target="_blank">
```

Comment:
Empty or "#" links aren't valid based on the rule https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-is-valid.md. Since we want to show the link behaviour in our examples, we use a placeholder anchor so that we don't navigate away from storybook!